### ISSUE TITLE
Failed to reshape the batch data

### ISSUE THREAD

**Issue by Ethan-Lea** [dariush-salami/attention-based-edge-convolution#1]
Nice work! But a problem of batch size setting occurred when running the code. To be specific, from the implementation of pytorch geometric, the data loader will return a batch of data with shape of [total node numbers in a batch, feature dimensions of each node]. And there is a line of code "batch_data = data.pos.cpu().detach().numpy().reshape((self.batch_size, -1, 3))" in class AugmentationTransformer which tries to reshape the batch data from [total node numbers in a batch, feature dimensions of each node] to [batch size,-1,feature dimensions of each node]. It is clear that the number of points in each gesture sample are different, so (total node numbers in a batch / batch size) is not always an integer, and the python interpreter will return an error like "ValueError: cannot reshape array of size xx into shape (xxx,newaxis,3)".
In short, I am confused about the reshape operation, and I would be grateful if you could give me some help.

**Comment by Haiyiwu-UCAS** [comment]
Hello, yadong, this is HaiyiWU from UCAS. I would like to try to reproduce this paper now, but since I am new to deep learning and the readme for this project is too brief, I see this huge pile of files and don't know where to start. I would like to ask if you could give me some of your previous route and experience in reproducing this project. Thank you very much!
